package.json

```diff
@@ -64,7 +64,7 @@
     "process": "PrismarineJS/node-process",
     "standard": "^16.0.3",
     "stream-browserify": "^3.0.0",
-    "three": "^0.128.0",
```

Comment:
Can you add back the ^ ?
It's not needed to remove it because the version is 0.x.y so 128 is a new major so ^ won't take it

Reply:
Fixed